### ISSUE TITLE
Visualize Problems

### ISSUE THREAD

**Issue by GaoXinJian-USTC** [pengzhiliang/MAE-pytorch#89]
I am surprised to find that it can reconstruct very well under the 95% mask ratio. So, I suspect whether an information leak occurs?

**Comment by pengzhiliang** [comment]
Yes, you are right.

**Comment by mouxinyue1** [comment]
Hello, I would like to ask if the weight file loaded visually is the pre-trained weight file or the fine-tuned weight file? Error when I load the fine-tuned weight file:
![image](https://user-images.githubusercontent.com/91049826/202651077-a4f6e3d8-2146-499a-87fa-745ba1e4b54b.png)
